playbooks/openstack/openshift-cluster/uninstall.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Unsubscribe RHEL instances
+  hosts: oo_all_hosts
```

Comment:
I should have noticed this earlier, sorry! The `oo_all_hosts` group comes from the `evaluate_groups.yml` playbook.

You must put this import before this task otherwise no hosts will be matched:

```yaml
- name: evaluate groups
  import_playbook: ../../init/evaluate_groups.yml
```

After that, all the nodes are matched and will be unsubscribed.